gamestonk_terminal/portfolio/portfolio_view.py

```diff
@@ -251,6 +251,39 @@ def plot_ef(
     return ImageReader(imgdata)
 
 
+def al(data: pd.DataFrame, graph: bool):
```

Comment:
This function name is a little ambiguous. I would rename to something like display_allocation

Reply:
Fixed :)